onivim2 build failure on macOS Catalina with macports

On a current macOS Catalina installation using macports, I am unable to build oni2. The build bombs out with:

https://gist.github.com/raw-bin/4290e6b75bf65df09d467dcf8f23e542

Something (perhaps a CMake include directive or some esy equivalent) appears to reference /usr/local/include as an include path which I believe is incorrect for a macports installation (which uses /opt/local/include).

For some reason, this results in a warning due to the compiler being built with '-Wpoison-system-directories'. That then bombs out because of '-Werror'.

As a data point: I managed to build one of the objects above in isolation by cutting and pasting the compiler invocation on the command line while taking care to remove '-Werror'. Not ideal of course. I couldn't figure out where to override the CFLAGS that are used for building objects in esy_libjpeg_turbo to remove '-Werror' globally. That would have given me a stop-gap build.

Any pointers appreciated. I'd rather not switch to Homebrew!

Thanks

I'm also experiencing a similar issue:

```
~/oni2 master ⇣ 2m 41s
❯ esy bootstrap
info building @esy-ocaml/substs@0.0.1@d41d8cd9
info building ocaml@4.10.0@d41d8cd9
info building yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9
info building esy-help2man@github:esy-packages/esy-help2man#c8e6931d1dcf58a81bd801145a777fd3b115c443@d41d8cd9
info building esy-angle-prebuilt@1.0.0@d41d8cd9
info building esy-cmake@0.3.5@d41d8cd9
info building esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9
info building esy-sdl2@2.0.10008@d41d8cd9
info building esy-angle-prebuilt@1.0.0@d41d8cd9: done
info building esy-cmake@0.3.5@d41d8cd9: done
info building @esy-cross/ninja-build@1.8.2001@d41d8cd9
info building @esy-ocaml/substs@0.0.1@d41d8cd9: done
info building @revery/esy-cmake@0.3.5001@d41d8cd9
info building esy-nasm@github:revery-ui/esy-nasm#64a802b@d41d8cd9
info building @esy-cross/ninja-build@1.8.2001@d41d8cd9: done
info building esy-tree-sitter@1.4.1@d41d8cd9
info building esy-nasm@github:revery-ui/esy-nasm#64a802b@d41d8cd9: done
info building libvim@8.10869.59@d41d8cd9
info building esy-help2man@github:esy-packages/esy-help2man#c8e6931d1dcf58a81bd801145a777fd3b115c443@d41d8cd9: done
info building @revery/esy-harfbuzz@2.6.8001@d41d8cd9
info building esy-tree-sitter@1.4.1@d41d8cd9: done
info building texinfo@github:esy-packages/esy-texinfo#4a05feafbbcc4c57d5d25899fbdab98961b9a69c@d41d8cd9
info building esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9: done
info building esy-freetype2@2.9.1008@d41d8cd9
info building esy-freetype2@2.9.1008@d41d8cd9: done
info building @opam/base-bigarray@opam:base@b03491b0
info building @opam/base-bigarray@opam:base@b03491b0: done
info building @opam/conf-m4@opam:1@3b2b148a
info building @opam/conf-m4@opam:1@3b2b148a: done
info building @opam/base-unix@opam:base@87d0b2eb
info building @opam/base-unix@opam:base@87d0b2eb: done
info building @opam/base-threads@opam:base@36803084
info building @opam/base-threads@opam:base@36803084: done
info building esy-macdylibbundler@0.4.5@d41d8cd9
info building autoconf@github:esy-packages/esy-autoconf#fb93edf57b0adc4b27b34a57a562395b224002d3@d41d8cd9
info building esy-macdylibbundler@0.4.5@d41d8cd9: done
info building esy-autoconf@github:esy-packages/esy-autoconf#fb93edf@d41d8cd9
info building autoconf@github:esy-packages/esy-autoconf#fb93edf57b0adc4b27b34a57a562395b224002d3@d41d8cd9: done
info building automake@github:esy-packages/esy-automake#e959059ccc1560a1565b16fb63ed7052c663fca0@d41d8cd9
info building esy-autoconf@github:esy-packages/esy-autoconf#fb93edf@d41d8cd9: done
info building @opam/conf-autoconf@opam:0.1@27b3f7cf
info building @opam/conf-autoconf@opam:0.1@27b3f7cf: done
info building automake@github:esy-packages/esy-automake#e959059ccc1560a1565b16fb63ed7052c663fca0@d41d8cd9: done
info building texinfo@github:esy-packages/esy-texinfo#4a05feafbbcc4c57d5d25899fbdab98961b9a69c@d41d8cd9: done
info building esy-libtools@github:revery-ui/esy-libtools#c9eb685@d41d8cd9
info building esy-libtools@github:revery-ui/esy-libtools#c9eb685@d41d8cd9: done
info building @revery/esy-libvterm@1.0.3@d41d8cd9
info building yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9: done
info building @opam/conf-pkg-config@opam:1.2@d86c8f53
info building @opam/conf-pkg-config@opam:1.2@d86c8f53: done
info building @reason-native-web/esy-openssl@1.1.1006@d41d8cd9
info building esy-sdl2@2.0.10008@d41d8cd9: done
info building ocaml@4.10.0@d41d8cd9: done
info building @opam/dune@opam:2.5.0@e0bac278
info building @opam/ocamlfind@opam:1.8.1@ff07b0f9
info building @opam/cmdliner@opam:1.0.4@93208aac
info building @opam/ocamlbuild@opam:0.14.0@6ac75d03
info building @opam/seq@opam:base@d8d7de1d
info building @opam/seq@opam:base@d8d7de1d: done
info building @opam/cmdliner@opam:1.0.4@93208aac: done
info building @opam/ocamlfind@opam:1.8.1@ff07b0f9: done
info building @opam/base-bytes@opam:base@19d0c2ff
info building @opam/base-bytes@opam:base@19d0c2ff: done
info building @opam/ocamlbuild@opam:0.14.0@6ac75d03: done
info building @opam/topkg@opam:1.0.3@e4e10f1c
info building @opam/uchar@opam:0.0.2@c8218eea
info building @opam/uchar@opam:0.0.2@c8218eea: done
info building @opam/topkg@opam:1.0.3@e4e10f1c: done
info building @opam/react@opam:1.2.1@0e11855f
info building @opam/astring@opam:0.8.5@1300cee8
info building @opam/uutf@opam:1.0.2@4440868f
info building @opam/uutf@opam:1.0.2@4440868f: done
info building @opam/uucp@opam:13.0.0@e9b515e0
info building @opam/react@opam:1.2.1@0e11855f: done
info building @opam/astring@opam:0.8.5@1300cee8: done
info building @opam/fpath@opam:0.7.3@674d8125
info building @opam/fpath@opam:0.7.3@674d8125: done
info building @opam/dune@opam:2.5.0@e0bac278: done
info building @opam/omd@github:ocaml/omd:omd.opam#1535e3c@d41d8cd9
info building @opam/cppo@opam:1.6.6@f4f83858
info building @opam/fix@opam:20200131@0ecd2f01
info building @opam/menhirLib@opam:20200624@8bdd2b0e
info building @opam/fix@opam:20200131@0ecd2f01: done
info building @opam/menhirSdk@opam:20200624@2a05b5a7
info building @opam/cppo@opam:1.6.6@f4f83858: done
info building @opam/ppx_derivers@opam:1.2.1@ecf0aa45
info building @opam/menhirLib@opam:20200624@8bdd2b0e: done
info building @opam/result@opam:1.5@6b753c82
info building @opam/ppx_derivers@opam:1.2.1@ecf0aa45: done
info building @opam/stdlib-shims@opam:0.1.0@8c116481
info building @opam/result@opam:1.5@6b753c82: done
info building @opam/dune-private-libs@opam:2.5.1@60c1661f
info building @opam/stdlib-shims@opam:0.1.0@8c116481: done
info building @opam/mmap@opam:1.1.0@b85334ff
info building @opam/menhirSdk@opam:20200624@2a05b5a7: done
info building @opam/re@opam:1.9.0@d4d5e13d
info building @opam/mmap@opam:1.1.0@b85334ff: done
info building @opam/bigarray-compat@opam:1.0.0@1faefa97
info building @opam/bigarray-compat@opam:1.0.0@1faefa97: done
info building @opam/camomile@opam:1.0.2@40411a6b
info building @opam/re@opam:1.9.0@d4d5e13d: done
info building @opam/decoders@opam:0.4.0@b66e1fb4
info building @opam/decoders@opam:0.4.0@b66e1fb4: done
info building @opam/easy-format@opam:1.3.2@0484b3c4
info building @opam/easy-format@opam:1.3.2@0484b3c4: done
info building @opam/psq@opam:0.2.0@247756d4
info building @opam/omd@github:ocaml/omd:omd.opam#1535e3c@d41d8cd9: done
info building @opam/integers@opam:0.4.0@f7acfaeb
info building @opam/psq@opam:0.2.0@247756d4: done
info building @opam/markup@opam:0.8.2@87975241
info building @opam/integers@opam:0.4.0@f7acfaeb: done
info building @opam/dune-build-info@opam:2.6.2@03ffe168
info building @opam/markup@opam:0.8.2@87975241: done
info building @opam/sexplib0@opam:v0.14.0@ddeb6438
info building @opam/dune-build-info@opam:2.6.2@03ffe168: done
info building @opam/ppx_tools@opam:6.2@62a3aff2
info building @opam/sexplib0@opam:v0.14.0@ddeb6438: done
info building @opam/ocaml-compiler-libs@opam:v0.12.1@5c34eb0d
info building @opam/dune-private-libs@opam:2.5.1@60c1661f: done
info building @opam/jane-street-headers@opam:v0.14.0@59432b6a
info building @opam/jane-street-headers@opam:v0.14.0@59432b6a: done
info building @opam/octavius@opam:1.2.2@b328d1f1
info building @opam/ppx_tools@opam:6.2@62a3aff2: done
info building @opam/trie@opam:1.0.0@d2efc587
info building @opam/trie@opam:1.0.0@d2efc587: done
info building @opam/ocplib-endian@opam:1.1@84c1ca88
info building @opam/ocaml-compiler-libs@opam:v0.12.1@5c34eb0d: done
info building @opam/merlin-extend@opam:0.6@404f814c
info building @opam/ocplib-endian@opam:1.1@84c1ca88: done
info building @opam/ocaml-migrate-parsetree@opam:1.7.3@dbcf3b47
info building @opam/octavius@opam:1.2.2@b328d1f1: done
info building @opam/rresult@opam:0.6.0@4b185e72
info building @opam/camomile@opam:1.0.2@40411a6b: done
info building @opam/ptime@opam:0.8.5@0051d642
info building @opam/merlin-extend@opam:0.6@404f814c: done
info building @opam/fmt@opam:0.8.8@01c3a23c
info building @opam/rresult@opam:0.6.0@4b185e72: done
info building @opam/menhir@opam:20200624@8629ff13
info building @opam/fmt@opam:0.8.8@01c3a23c: done
info building @opam/bigstringaf@opam:0.6.1@35f5e6d1
info building @opam/ptime@opam:0.8.5@0051d642: done
info building @opam/tyxml@opam:4.4.0@1dca5713
info building @opam/bigstringaf@opam:0.6.1@35f5e6d1: done
info building @opam/uucp@opam:13.0.0@e9b515e0: done
info building @opam/biniou@opam:1.2.1@d7570399
info building @opam/lru@github:bryphe/lru:lru.opam#2708c70@d41d8cd9
info building @opam/lru@github:bryphe/lru:lru.opam#2708c70@d41d8cd9: done
info building @opam/biniou@opam:1.2.1@d7570399: done
info building @opam/ctypes@opam:0.15.1@b0227b2f
info building @opam/dune-configurator@opam:2.5.1@aeb9d8d5
info building @opam/tyxml@opam:4.4.0@1dca5713: done
info building @opam/mew@opam:0.1.0@a74f69d6
info building @opam/mew@opam:0.1.0@a74f69d6: done
info building @opam/charInfo_width@opam:1.1.0@4296bdfe
info building @opam/charInfo_width@opam:1.1.0@4296bdfe: done
info building @opam/angstrom@opam:0.14.1@07e286b0
info building @opam/angstrom@opam:0.14.1@07e286b0: done
info building @opam/uuseg@opam:13.0.0@f60712a7
info building @opam/menhir@opam:20200624@8629ff13: done
info building @opam/yojson@github:onivim/yojson:yojson.opam#f480aef@d41d8cd9
info building @opam/dune-configurator@opam:2.5.1@aeb9d8d5: done
info building @opam/junit@opam:2.0.2@0b7bd730
info building @opam/junit@opam:2.0.2@0b7bd730: done
info building @opam/odoc@opam:1.5.1@52a58c0b
info building @opam/ocaml-migrate-parsetree@opam:1.7.3@dbcf3b47: done
info building @opam/printbox@opam:0.5@82f5d436
info building @opam/uuseg@opam:13.0.0@f60712a7: done
info building @opam/mew_vi@opam:0.5.0@cf66c299
info building @opam/yojson@github:onivim/yojson:yojson.opam#f480aef@d41d8cd9: done
info building @opam/zed@opam:3.1.0@86c55416
info building @opam/printbox@opam:0.5@82f5d436: done
info building @opam/atd@opam:2.2.1@071ab6bd
info building @opam/mew_vi@opam:0.5.0@cf66c299: done
info building @esy-ocaml/reason@3.6.2@d41d8cd9
info building @opam/zed@opam:3.1.0@86c55416: done
info building @opam/lwt@opam:4.5.0@542100aa
info building @opam/atd@opam:2.2.1@071ab6bd: done
info building reason-native-crash-utils@github:onivim/reason-native-crash-utils#38c8f00@d41d8cd9
info building reason-native-crash-utils@github:onivim/reason-native-crash-utils#38c8f00@d41d8cd9: done
info building @onivim/reason-native-crash-utils@1.0.1@d41d8cd9
info building @onivim/reason-native-crash-utils@1.0.1@d41d8cd9: done
info building @opam/base@opam:v0.14.0@8bc55fce
info building @opam/lwt@opam:4.5.0@542100aa: done
info building @opam/reason@opam:3.6.0@58adb39a
info building @opam/ctypes@opam:0.15.1@b0227b2f: done
info building @opam/ppxlib@opam:0.15.0@6a9d8126
info building @opam/odoc@opam:1.5.1@52a58c0b: done
info building @opam/ppx_tools_versioned@opam:5.4.0@32d1a414
info building @opam/base@opam:v0.14.0@8bc55fce: done
info building @opam/ppxfind@opam:1.4@d75848d2
info building @opam/ppx_tools_versioned@opam:5.4.0@32d1a414: done
info building @opam/atdgen-runtime@opam:2.2.1@6a3a6395
info building @opam/ppxfind@opam:1.4@d75848d2: done
info building @opam/decoders-yojson@opam:0.4.0@cb39dea6
info building @opam/atdgen-runtime@opam:2.2.1@6a3a6395: done
info building @opam/ppx_yojson_conv_lib@opam:v0.14.0@116b53d6
info building @opam/decoders-yojson@opam:0.4.0@cb39dea6: done
info building @opam/logs@opam:0.7.0@1d03143e
info building @opam/ppx_yojson_conv_lib@opam:v0.14.0@116b53d6: done
info building @opam/lwt_react@opam:1.1.3@72987fcf
info building @opam/lwt_react@opam:1.1.3@72987fcf: done
info building @opam/lwt_log@opam:1.1.1@2d7a797f
info building @opam/lwt_log@opam:1.1.1@2d7a797f: done
info building @opam/luv@github:bryphe/luv:luv.opam#207d3f6@d41d8cd9
info building @opam/logs@opam:0.7.0@1d03143e: done
info building @opam/stdio@opam:v0.14.0@a624e254
info building @opam/stdio@opam:v0.14.0@a624e254: done
info building @opam/lwt_ppx@opam:2.0.1@e6a764a0
info building @esy-ocaml/reason@3.6.2@d41d8cd9: done
info building @opam/ppx_deriving@opam:4.5@d89f2934
info building @opam/lwt_ppx@opam:2.0.1@e6a764a0: done
info building @opam/atdgen@opam:2.2.1@d73fda11
info building @opam/reason@opam:3.6.0@58adb39a: done
info building @opam/ocaml-lsp-server@github:ocaml/ocaml-lsp:ocaml-lsp-server.opam#c1e51a5@d41d8cd9
info building @opam/ppx_deriving@opam:4.5@d89f2934: done
info building @opam/ppxlib@opam:0.15.0@6a9d8126: done
info building @opam/atdgen@opam:2.2.1@d73fda11: done
info building @opam/lambda-term@opam:3.1.0@8adc2660
info building @opam/bos@opam:0.2.0@df49e63f
info building @opam/ocamlformat@opam:0.15.0@8e036963
info building @reason-native-web/esy-openssl@1.1.1006@d41d8cd9: done
info building rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9
info building rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9: done
info building flex@github:bryphe/flex#5e19b05@d41d8cd9
info building @opam/bos@opam:0.2.0@df49e63f: done
info building @reason-native/pastel@0.1.0@d41d8cd9
info building flex@github:bryphe/flex#5e19b05@d41d8cd9: done
info building @reason-native/console@0.0.3@d41d8cd9
info building @reason-native/pastel@0.1.0@d41d8cd9: done
info building @glennsl/timber@1.2.0@d41d8cd9
info building @reason-native/console@0.0.3@d41d8cd9: done
info building @opam/lambda-term@opam:3.1.0@8adc2660: done
info building @opam/fp@github:bryphe/reason-native:fp.opam#fd0225c@d41d8cd9
info building @brisk/brisk-reconciler@github:briskml/brisk-reconciler#10cab2d@d41d8cd9
info building @opam/luv@github:bryphe/luv:luv.opam#207d3f6@d41d8cd9: done
info building @opam/ppx_let@opam:v0.14.0@eb9b93e0
info building @glennsl/timber@1.2.0@d41d8cd9: done
info building @opam/ppx_optcomp@opam:v0.14.0@d77a04c2
info building @opam/fp@github:bryphe/reason-native:fp.opam#fd0225c@d41d8cd9: done
info building @opam/ppx_js_style@opam:v0.14.0@10b020a8
info building @opam/ppx_let@opam:v0.14.0@eb9b93e0: done
info building @opam/ppx_enumerate@opam:v0.14.0@63db8245
info building @opam/ppx_optcomp@opam:v0.14.0@d77a04c2: done
info building @opam/ppx_sexp_conv@opam:v0.14.1@a205b550
info building @revery/esy-harfbuzz@2.6.8001@d41d8cd9: done
info building @opam/ppx_here@opam:v0.14.0@5ccc1c01
info building @brisk/brisk-reconciler@github:briskml/brisk-reconciler#10cab2d@d41d8cd9: done
info building @opam/ppx_compare@opam:v0.14.0@615de7a6
info building @opam/ppx_js_style@opam:v0.14.0@10b020a8: done
info building @opam/ppx_cold@opam:v0.14.0@345dec7c
info building @opam/ppx_enumerate@opam:v0.14.0@63db8245: done
info building @opam/ppx_deriving_yojson@opam:3.5.3@253e094d
info building @opam/ppx_here@opam:v0.14.0@5ccc1c01: done
info building @reason-native/cli@0.0.1-alpha@d41d8cd9
info building @reason-native/cli@0.0.1-alpha@d41d8cd9: done
info building @reason-native/file-context-printer@0.0.3@d41d8cd9
info building @opam/ocamlformat@opam:0.15.0@8e036963: done
info building refmterr@3.3.2@d41d8cd9
info building @opam/ppx_cold@opam:v0.14.0@345dec7c: done
info building @opam/fs@github:bryphe/reason-native:fs.opam#fd0225c@d41d8cd9
info building @opam/ppx_deriving_yojson@opam:3.5.3@253e094d: done
info building @reason-native/file-context-printer@0.0.3@d41d8cd9: done
info building @opam/ppx_compare@opam:v0.14.0@615de7a6: done
info building @opam/ppx_sexp_conv@opam:v0.14.1@a205b550: done
info building @opam/dir@github:bryphe/reason-native:dir.opam#fd0225@d41d8cd9
info building @opam/ppx_hash@opam:v0.14.0@8e9618e4
info building @opam/ppx_assert@opam:v0.14.0@877b5900
info building @reason-native/rely@3.2.1@d41d8cd9
info building refmterr@3.3.2@d41d8cd9: done
info building reperf@1.5.0@d41d8cd9
info building @opam/dir@github:bryphe/reason-native:dir.opam#fd0225@d41d8cd9: done
info building @opam/fs@github:bryphe/reason-native:fs.opam#fd0225c@d41d8cd9: done
info building @opam/ppx_hash@opam:v0.14.0@8e9618e4: done
info building @reason-native/rely@3.2.1@d41d8cd9: done
info building isolinear@github:revery-ui/isolinear#53fc4eb@d41d8cd9
info building @opam/ppx_base@opam:v0.14.0@b4702ed9
info building rench@github:bryphe/rench#a976fe5@d41d8cd9
info building @opam/ocaml-lsp-server@github:ocaml/ocaml-lsp:ocaml-lsp-server.opam#c1e51a5@d41d8cd9: done
info building @opam/ppx_assert@opam:v0.14.0@877b5900: done
info building @opam/jst-config@opam:v0.14.0@d0d7469e
info building reperf@1.5.0@d41d8cd9: done
info building isolinear@github:revery-ui/isolinear#53fc4eb@d41d8cd9: done
info building reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9
info building rench@github:bryphe/rench#a976fe5@d41d8cd9: done
info building reasonFuzz@github:CrossR/reasonFuzz#1ad6f5d@d41d8cd9: done
info building @opam/ppx_base@opam:v0.14.0@b4702ed9: done
info building @opam/jst-config@opam:v0.14.0@d0d7469e: done
info building @opam/time_now@opam:v0.14.0@5e4046b3
info building @opam/time_now@opam:v0.14.0@5e4046b3: done
info building @opam/ppx_inline_test@opam:v0.14.1@2e4fdd8d
info building @opam/ppx_inline_test@opam:v0.14.1@2e4fdd8d: done
info building @revery/esy-cmake@0.3.5001@d41d8cd9: done
info building esy-libjpeg-turbo@github:revery-ui/libjpeg-turbo#dbb3dd5@d41d8cd9
info building esy-libjpeg-turbo@github:revery-ui/libjpeg-turbo#dbb3dd5@d41d8cd9: done
info building esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9
error: build failed with exit code: 1
  build log:
    # esy-build-package: building: esy-skia@github:revery-ui/esy-skia#a3785f9
    # esy-build-package: pwd: /Users/holmes/.esy/3/b/esy_skia-b4348b22
    # esy-build-package: running: 'bash' './esy/build.sh' 'darwin' '/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679'
    buildtools                           @ 505de88083136eefd056e5ee4ca0f01fe9b33de8
    common                               @ 9737551d7a52c3db3262db5856e6bcd62c462b92
    third_party/externals/angle2         @ b001528ffa00e7c15a5002124f707570e59a5697
    third_party/externals/dng_sdk        @ 96443b262250c390b0caefbf3eed8463ba35ecae
    third_party/externals/expat          @ android-6.0.1_r55
    third_party/externals/freetype       @ fa6da7bf6dcbcfa22d2464259ad1cccee3233189
    third_party/externals/googletest     @ dd43b9998e9a44a579a7aba6c1309407d1a5ed95
    third_party/externals/harfbuzz       @ 1.4.2
    third_party/externals/icu            @ ec9c1133693148470ffe2e5e53576998e3650c1d
    third_party/externals/imgui          @ 6384eee34f08cb7eab8d835043e1738e4adcdf75
    third_party/externals/jsoncpp        @ 1.0.0
    third_party/externals/libjpeg-turbo  @ 1.5.3
    third_party/externals/libpng         @ v1.6.33
    third_party/externals/libwebp        @ v0.6.1
    third_party/externals/lua            @ v5-3-4
    third_party/externals/microhttpd     @ 748945ec6f1c67b7efc934ab0808e1d32f2fb98d
    third_party/externals/piex           @ bb217acdca1cc0c16b704669dd6f91a1b509c406
    third_party/externals/rapidjson      @ af223d44f4e8d3772cb1ac0ce8bc2a132b51717f
    third_party/externals/sdl            @ 5d7cfcca344034aff9327f77fc181ae3754e7a90
    third_party/externals/sfntly         @ b18b09b6114b9b7fe6fc2f96d8b15e8a72f66916
    third_party/externals/spirv-headers  @ 661ad91124e6af2272afd00f804d8aa276e17107
    third_party/externals/spirv-tools    @ e9e4393b1c5aad7553c05782acefbe32b42644bd
    third_party/externals/swiftshader    @ cbb80f5f0078a9941f3ec43e83e52c3d15a43bea
    third_party/externals/zlib           @ e7afdfe128e01ca480a28f757b571957befdd962
    llvm toolset-7.0 does not need to be manually activated
    Done. Made 42 targets from 25 files in 75ms
    ninja: Entering directory `/Users/holmes/.esy/3/b/esy_skia-b4348b22/out/Static'
    [1/1196] stamp obj/armv7.stamp
    [2/1196] stamp obj/arm64.stamp
    [3/1196] stamp obj/compile_processors.stamp
    [4/1196] stamp obj/crc32.stamp
    [5/1196] compile ../../src/effects/SkColorMatrix.cpp
    FAILED: obj/src/effects/effects.SkColorMatrix.o 
    clang++ -MMD -MF obj/src/effects/effects.SkColorMatrix.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/SkColorMatrix.cpp -o obj/src/effects/effects.SkColorMatrix.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [6/1196] compile ../../src/effects/Sk1DPathEffect.cpp
    FAILED: obj/src/effects/effects.Sk1DPathEffect.o 
    clang++ -MMD -MF obj/src/effects/effects.Sk1DPathEffect.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/Sk1DPathEffect.cpp -o obj/src/effects/effects.Sk1DPathEffect.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [7/1196] compile ../../src/effects/Sk2DPathEffect.cpp
    FAILED: obj/src/effects/effects.Sk2DPathEffect.o 
    clang++ -MMD -MF obj/src/effects/effects.Sk2DPathEffect.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/Sk2DPathEffect.cpp -o obj/src/effects/effects.Sk2DPathEffect.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [8/1196] compile ../../src/effects/SkCornerPathEffect.cpp
    FAILED: obj/src/effects/effects.SkCornerPathEffect.o 
    clang++ -MMD -MF obj/src/effects/effects.SkCornerPathEffect.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/SkCornerPathEffect.cpp -o obj/src/effects/effects.SkCornerPathEffect.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [9/1196] compile ../../src/effects/SkComposeImageFilter.cpp
    FAILED: obj/src/effects/effects.SkComposeImageFilter.o 
    clang++ -MMD -MF obj/src/effects/effects.SkComposeImageFilter.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/SkComposeImageFilter.cpp -o obj/src/effects/effects.SkComposeImageFilter.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [10/1196] compile ../../src/effects/SkColorFilterImageFilter.cpp
    FAILED: obj/src/effects/effects.SkColorFilterImageFilter.o 
    clang++ -MMD -MF obj/src/effects/effects.SkColorFilterImageFilter.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/SkColorFilterImageFilter.cpp -o obj/src/effects/effects.SkColorFilterImageFilter.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [11/1196] compile ../../src/effects/SkColorMatrixFilter.cpp
    FAILED: obj/src/effects/effects.SkColorMatrixFilter.o 
    clang++ -MMD -MF obj/src/effects/effects.SkColorMatrixFilter.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/SkColorMatrixFilter.cpp -o obj/src/effects/effects.SkColorMatrixFilter.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [12/1196] compile ../../src/opts/SkOpts_avx.cpp
    FAILED: obj/src/opts/avx.SkOpts_avx.o 
    clang++ -MMD -MF obj/src/opts/avx.SkOpts_avx.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -mavx -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/opts/SkOpts_avx.cpp -o obj/src/opts/avx.SkOpts_avx.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [13/1196] compile ../../src/effects/SkAlphaThresholdFilter.cpp
    FAILED: obj/src/effects/effects.SkAlphaThresholdFilter.o 
    clang++ -MMD -MF obj/src/effects/effects.SkAlphaThresholdFilter.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/SkAlphaThresholdFilter.cpp -o obj/src/effects/effects.SkAlphaThresholdFilter.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    [14/1196] compile ../../src/effects/SkArithmeticImageFilter.cpp
    FAILED: obj/src/effects/effects.SkArithmeticImageFilter.o 
    clang++ -MMD -MF obj/src/effects/effects.SkArithmeticImageFilter.o.d -DNDEBUG -DSK_GAMMA_APPLY_TO_A8 -DSK_ENABLE_DISCRETE_GPU -DGR_TEST_UTILS=1 -DSKIA_IMPLEMENTATION=1 -I../../include/android -I../../include/c -I../../include/codec -I../../include/config -I../../include/core -I../../include/effects -I../../include/encode -I../../include/gpu -I../../include/atlastext -I../../include/pathops -I../../include/ports -I../../include/svg -I../../include/utils -I../../include/utils/mac -I../../include/private -I../../src/c -I../../src/codec -I../../src/core -I../../src/effects -I../../src/fonts -I../../src/image -I../../src/images -I../../src/lazy -I../../src/opts -I../../src/pathops -I../../src/pdf -I../../src/ports -I../../src/sfnt -I../../src/shaders -I../../src/shaders/gradients -I../../src/sksl -I../../src/utils -I../../src/utils/win -I../../src/xml -I../../third_party/gif -I../../src/gpu -fstrict-aliasing -fPIC -fvisibility=hidden -O3 -fdata-sections -ffunction-sections -g -Werror -Wall -Wextra -Winit-self -Wpointer-arith -Wsign-compare -Wvla -Wno-deprecated-declarations -Wno-maybe-uninitialized -Wno-extra-semi-stmt -Weverything -Wno-unknown-warning-option -Wno-anon-enum-enum-conversion -Wno-sizeof-array-div -Wno-nonportable-include-path -Wno-nonportable-system-include-path -Wno-cast-align -Wno-cast-qual -Wno-conversion -Wno-disabled-macro-expansion -Wno-documentation -Wno-documentation-unknown-command -Wno-double-promotion -Wno-exit-time-destructors -Wno-float-equal -Wno-format-nonliteral -Wno-global-constructors -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-pedantic -Wno-reserved-id-macro -Wno-shadow -Wno-shift-sign-overflow -Wno-signed-enum-bitfield -Wno-switch-enum -Wno-undef -Wno-unreachable-code -Wno-unreachable-code-break -Wno-unreachable-code-return -Wno-unused-macros -Wno-unused-member-function -Wno-unused-template -Wno-zero-as-null-pointer-constant -Wno-bad-function-cast -Wno-covered-switch-default -Wno-deprecated -Wno-missing-noreturn -Wno-old-style-cast -Wno-padded -Wno-newline-eof -Wno-implicit-fallthrough -Wno-unused-parameter -I/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679/include -std=c++11 -fvisibility-inlines-hidden -fno-exceptions -fno-rtti -Wnon-virtual-dtor -Wno-noexcept-type -Wno-abstract-vbase-init -Wno-weak-vtables -Wno-c++98-compat -Wno-c++98-compat-pedantic -Wno-undefined-func-template -c ../../src/effects/SkArithmeticImageFilter.cpp -o obj/src/effects/effects.SkArithmeticImageFilter.o
    error: include location '/usr/local/include' is unsafe for cross-compilation [-Werror,-Wpoison-system-directories]
    1 error generated.
    ninja: build stopped: subcommand failed.
    error: command failed: 'bash' './esy/build.sh' 'darwin' '/Users/holmes/.esy/3__________________________________________________________________/i/esy_libjpeg_turbo-dc3b4679' (exited with 1)
    esy-build-package: exiting with errors above...
    
  building esy-skia@github:revery-ui/esy-skia#a3785f9
esy: exiting due to errors above
```

I'm running macOS Catalina, and I have all the dependencies installed as outlined in https://onivim.github.io/docs/for-developers/building.

@raw-bin Did you discover any resolutions?

--- a/bench.esy.lock/index.json
+++ b/bench.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "6c75cff7f6f58bccf8419b7648733903",
+  "checksum": "486019be99d740bdc109fbe369502a25",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -141,7 +141,7 @@
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#38c8f00@d41d8cd9",
         "flex@github:bryphe/flex#5e19b05@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9", "esy-freetype2@2.9.1008@d41d8cd9",
         "esy-angle-prebuilt@1.0.0@d41d8cd9",
         "@revery/esy-harfbuzz@2.6.8001@d41d8cd9",
@@ -630,13 +630,13 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9": {
-      "id": "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+    "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9": {
+      "id": "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
       "name": "esy-skia",
-      "version": "github:revery-ui/esy-skia#a3785f9",
+      "version": "github:revery-ui/esy-skia#60e0260",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/esy-skia#a3785f9" ]
+        "source": [ "github:revery-ui/esy-skia#60e0260" ]
       },
       "overrides": [],
       "dependencies": [
@@ -945,7 +945,7 @@
         "ocaml@4.10.0@d41d8cd9", "libvim@8.10869.61@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#53fc4eb@d41d8cd9",
         "esy-tree-sitter@1.4.1@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9",
         "esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9",

--- a/esy.lock/index.json
+++ b/esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "6c75cff7f6f58bccf8419b7648733903",
+  "checksum": "486019be99d740bdc109fbe369502a25",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -141,7 +141,7 @@
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#38c8f00@d41d8cd9",
         "flex@github:bryphe/flex#5e19b05@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9", "esy-freetype2@2.9.1008@d41d8cd9",
         "esy-angle-prebuilt@1.0.0@d41d8cd9",
         "@revery/esy-harfbuzz@2.6.8001@d41d8cd9",
@@ -630,13 +630,13 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9": {
-      "id": "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+    "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9": {
+      "id": "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
       "name": "esy-skia",
-      "version": "github:revery-ui/esy-skia#a3785f9",
+      "version": "github:revery-ui/esy-skia#60e0260",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/esy-skia#a3785f9" ]
+        "source": [ "github:revery-ui/esy-skia#60e0260" ]
       },
       "overrides": [],
       "dependencies": [
@@ -944,7 +944,7 @@
         "ocaml@4.10.0@d41d8cd9", "libvim@8.10869.61@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#53fc4eb@d41d8cd9",
         "esy-tree-sitter@1.4.1@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9",
         "esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9",

--- a/integrationtest.esy.lock/index.json
+++ b/integrationtest.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "6c75cff7f6f58bccf8419b7648733903",
+  "checksum": "486019be99d740bdc109fbe369502a25",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -141,7 +141,7 @@
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#38c8f00@d41d8cd9",
         "flex@github:bryphe/flex#5e19b05@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9", "esy-freetype2@2.9.1008@d41d8cd9",
         "esy-angle-prebuilt@1.0.0@d41d8cd9",
         "@revery/esy-harfbuzz@2.6.8001@d41d8cd9",
@@ -630,13 +630,13 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9": {
-      "id": "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+    "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9": {
+      "id": "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
       "name": "esy-skia",
-      "version": "github:revery-ui/esy-skia#a3785f9",
+      "version": "github:revery-ui/esy-skia#60e0260",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/esy-skia#a3785f9" ]
+        "source": [ "github:revery-ui/esy-skia#60e0260" ]
       },
       "overrides": [],
       "dependencies": [
@@ -944,7 +944,7 @@
         "ocaml@4.10.0@d41d8cd9", "libvim@8.10869.61@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#53fc4eb@d41d8cd9",
         "esy-tree-sitter@1.4.1@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9",
         "esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9",

--- a/package.json
+++ b/package.json
@@ -257,7 +257,7 @@
   },
   "resolutions": {
     "revery": "revery-ui/revery#18eb60e",
-    "esy-skia": "revery-ui/esy-skia#a3785f9",
+    "esy-skia": "revery-ui/esy-skia#60e0260",
     "rench": "bryphe/rench#a976fe5",
     "reasonFuzz": "CrossR/reasonFuzz#1ad6f5d",
     "isolinear": "revery-ui/isolinear#53fc4eb",

--- a/release.esy.lock/index.json
+++ b/release.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "6c75cff7f6f58bccf8419b7648733903",
+  "checksum": "486019be99d740bdc109fbe369502a25",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -141,7 +141,7 @@
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#38c8f00@d41d8cd9",
         "flex@github:bryphe/flex#5e19b05@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9", "esy-freetype2@2.9.1008@d41d8cd9",
         "esy-angle-prebuilt@1.0.0@d41d8cd9",
         "@revery/esy-harfbuzz@2.6.8001@d41d8cd9",
@@ -630,13 +630,13 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9": {
-      "id": "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+    "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9": {
+      "id": "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
       "name": "esy-skia",
-      "version": "github:revery-ui/esy-skia#a3785f9",
+      "version": "github:revery-ui/esy-skia#60e0260",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/esy-skia#a3785f9" ]
+        "source": [ "github:revery-ui/esy-skia#60e0260" ]
       },
       "overrides": [],
       "dependencies": [
@@ -944,7 +944,7 @@
         "ocaml@4.10.0@d41d8cd9", "libvim@8.10869.61@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#53fc4eb@d41d8cd9",
         "esy-tree-sitter@1.4.1@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9",
         "esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9",

--- a/test.esy.lock/index.json
+++ b/test.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "c0e8aeb211c0eee6e32328bdfe16efaf",
+  "checksum": "012c480fe36ede7be6919203c2da8857",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "yarn-pkg-config@github:esy-ocaml/yarn-pkg-config#db3a0b63883606dd57c54a7158d560d6cba8cd79@d41d8cd9": {
@@ -141,7 +141,7 @@
         "rebez@github:jchavarri/rebez#03fa3b7@d41d8cd9",
         "reason-native-crash-utils@github:onivim/reason-native-crash-utils#38c8f00@d41d8cd9",
         "flex@github:bryphe/flex#5e19b05@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9", "esy-freetype2@2.9.1008@d41d8cd9",
         "esy-angle-prebuilt@1.0.0@d41d8cd9",
         "@revery/esy-harfbuzz@2.6.8001@d41d8cd9",
@@ -630,13 +630,13 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9": {
-      "id": "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+    "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9": {
+      "id": "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
       "name": "esy-skia",
-      "version": "github:revery-ui/esy-skia#a3785f9",
+      "version": "github:revery-ui/esy-skia#60e0260",
       "source": {
         "type": "install",
-        "source": [ "github:revery-ui/esy-skia#a3785f9" ]
+        "source": [ "github:revery-ui/esy-skia#60e0260" ]
       },
       "overrides": [],
       "dependencies": [
@@ -944,7 +944,7 @@
         "ocaml@4.10.0@d41d8cd9", "libvim@8.10869.61@d41d8cd9",
         "isolinear@github:revery-ui/isolinear#53fc4eb@d41d8cd9",
         "esy-tree-sitter@1.4.1@d41d8cd9",
-        "esy-skia@github:revery-ui/esy-skia#a3785f9@d41d8cd9",
+        "esy-skia@github:revery-ui/esy-skia#60e0260@d41d8cd9",
         "esy-sdl2@2.0.10008@d41d8cd9",
         "esy-oniguruma@github:onivim/esy-oniguruma#4698ce4@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9",

